apps/web/pages/booking/[uid].tsx

```diff
@@ -278,7 +278,7 @@ export default function Success(props: SuccessProps) {
 
   // This is a weird case where the same route can be opened in booking flow as a success page or as a booking detail page from the app
   // As Booking Page it has to support configured theme, but as booking detail page it should not do any change. Let Shell.tsx handle it.
-  useTheme(isSuccessBookingPage ? props.profile.theme : undefined);
+  useTheme(isSuccessBookingPage ? props.profile.theme : "system");
```

Comment:
useTheme now explicilty needs "system" to be passed to support system theme instead of applying system theme when undefined or no argument is passed. This is more intuitive and less error prone.